utilities/src/main/java/io/syndesis/qe/bdd/validation/ServiceNowValidationSteps.java

```diff
@@ -0,0 +1,49 @@
+package io.syndesis.qe.bdd.validation;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.math.BigInteger;
+import java.util.UUID;
+
+import cucumber.api.java.en.When;
+import io.syndesis.qe.servicenow.ServiceNow;
+import io.syndesis.qe.servicenow.model.Incident;
+import io.syndesis.qe.utils.JMSUtils;
+
+public class ServiceNowValidationSteps {
+    private static String incidentId;
+
+    @When("^create incident with \"([^\"]*)\" number$")
+    public void createIncidentWithNumber(String number) {
+        Incident i = Incident.getSampleIncident();
+        i.setNumber(number);
+        i.setDescription("Automated create incident test-updated");
+        i.setSeverity(BigInteger.valueOf(2));
+        incidentId = ServiceNow.createIncident(i).getSysId();
+
+    }
+
+    @When("^verify that received incident from \"([^\"]*)\" queue contains \"([^\"]*)\"$")
+    public void verifyThatReceivedIncidentFromQueueContains(String queue, String contains) throws Throwable {
+        String message = JMSUtils.getMessageText(JMSUtils.Destination.valueOf("QUEUE"), queue);
+        System.out.println(message);
+        ServiceNow.deleteIncident(incidentId);
+        assertThat(message).contains(contains);
+        assertThat(message).contains(incidentId);
+    }
+
+    @When("^send \"([^\"]*)\" incident to \"([^\"]*)\" queue and verify it was created in SN$")
+    public void sendIncidentToQueueAndVerifyItWasCreatedInSN(String incidentNumber, String queue) throws Throwable {
+        final String description = "CRTDINC01" + UUID.randomUUID().toString().substring(0, 8);
+        Incident i = Incident.getSampleIncident();
+        i.setNumber(incidentNumber);
+        i.setDescription(description);
+        JMSUtils.sendMessage(JMSUtils.Destination.valueOf("QUEUE"), queue, ServiceNow.getIncidentJson(i));
+        //wait till message gets through integration
+        Thread.sleep(3000);
```

Comment:
use TestUtils.sleepForJenkinsDelayIfHigher(3) just in case, everything is slower in tests :/